.cookiecutter/cookiecutter.json

```diff
@@ -6,9 +6,11 @@
     "Dockerfile",
     "lms/app.py",
     "tests/unit/lms/app_test.py",
-    "lms/pshell.py",
+    "lms/db.py",
     "lms/migrations/env.py",
     "tests/functional/app_test.py",
+    "tests/pytest_plugins/db.py",
+    "tests/functional/pytest_plugins/db.py",
```

Comment:
LMS has its own test DB stuff, doesn't want to use the ones from the cookiecutter (at least not yet anyway)